src/main/java/tech/jhipster/lite/generator/client/angular/security/oauth2/domain/AngularOauth2ModuleFactory.java

```diff
@@ -56,14 +56,11 @@ public class AngularOauth2ModuleFactory {
   private static final String TEST_IMPORTS =
     """
     import { By } from '@angular/platform-browser';
-    import LoginComponent from './login/login.component';
-    """;
+    import LoginComponent from './login/login.component';""";
```

Comment:
FYI you can also add a backslash for avoiding line break at end:
```suggestion
    import LoginComponent from './login/login.component';\
""";
```